k3s-epinio-1-node/chartmuseum/install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+figlet chartmuseum
+
+export MY_IP=$(ip a s eth0 | egrep -o 'inet [0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | cut -d' ' -f2)
```

Comment:
Could be a good idea before to find the device, not sure if it's always `eth0`.

In that case you could do something like:
```
ETH_DEV=$(ip route | awk '/default via / { print $5 }')
```

This will get the device name of the default interface, should do the jon.